Increase imagenet_utils test coverage to 100%

tleeuwenburg opened the pull request:
Check a couple of code paths that weren't being followed

fchollet commented on tests/keras/applications/imagenet_utils_test.py:
```diff
@@ -37,8 +37,29 @@ def test_obtain_input_shape():
+        default_size=None,
+        min_size=139,
+        data_format='channels_first',
+        include_top=True,)
```
Why not test correctness?

fchollet commented on tests/keras/applications/imagenet_utils_test.py:
```diff
@@ -37,8 +37,29 @@ def test_obtain_input_shape():
+                min_size=139,
+                data_format=data_format,
+                include_top=False,
+                weights="fake_weights")
```
Please use single quotes for consistency

tleeuwenburg commented:
Sorry I polluted the history here in a way I didn't expect when I rebased. I didn't realise the things I had rebased against would appear in the review in this way. I'm going to close the merge request and open a new one with only the relevant changes.
